.travis.yml

```diff
@@ -75,3 +75,4 @@ script:
   - ./bin/dxc -T ps_6_0 -Fo passthru-ps.spv ../tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv -spirv
   - cmp passthru-ps.spv ../tools/clang/test/CodeGenSPIRV/passthru-ps.spv
   - ./bin/clang-spirv-tests --spirv-test-root ../tools/clang/test/CodeGenSPIRV/
+  - ./bin/clang-hlsl-tests --HlslDataDir $PWD/../tools/clang/test/CodeGenSPIRV/
```

Comment:
Shouldn't this be `$PWD/../tools/clang/test/CodeGenHLSL/` ? and if so, why hasn't the test failed? maybe AllocatorTest  doesn't use any hlsl input files?

Reply:
The expected location for that argument is that directory. All the HLSL tests, for any .hlsl files in CodeGenHLSL, will get there via "../CodeGenHLSL/<filename>.hlsl" In the case of allocatortest, it doesn't use any hlsl files, so it doesn't care at all, but there's just one invocation and other tests do expect the .hlsl files they need to be where HlslDataDir points, and those .hlsl files are found in /HLSL/ not /CodeGenHLSL/.

Reply:
You mean it'll find the .hlsl files by doing
`$PWD/../tools/clang/test/CodeGenSPIRV/../CodeGenHLSL/*.hlsl` ?

What if there was no CodeGenSPIRV directory? In my mind one should be able to run the HLSL tests even if there are no CodeGenSPIRV tests.

I find it very odd that `--HlslDataDir` points to `CodeGenSPIRV`... 

Am I missing something?

Reply:
Oh shoot. I see what you're saying. I didn't realize I'd included CodeGenSPIRV instead of HLSL. careless copy and paste.